specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1188,6 +1188,180 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/dedicatedSQLminimalTlsSettings/{dedicatedSQLminimalTlsSettingsName}": {
```

Comment:
>dedicatedSQLminimalTlsSettings [](http://example.com/codeflow?start=127&length=30)

why is this a child resource instead of a property on the workspace?  Users won't be able to perform this configuration until the workspace is created using this approach

Reply:
@pilor : We are setting Sql Server endpoint TLS version among other endpoints provided by workspace like SQL OD, Synapse GateWay.